src/routes/[...lang=locale]/anime/+page.svelte

```diff
@@ -299,123 +298,71 @@
-				<img
-					class="absolute h-full w-full rounded-[0.875vw] object-cover object-center"
-					src="https://i.pinimg.com/originals/b7/25/12/b725125aaebafbcbf2fb3886a55d2d6f.jpg"
-					alt=""
-					loading="lazy"
-				/>
-			</div>
-			<div class="absolute inset-0 bg-gradient-to-t from-secondary from-[1%] to-secondary/25"></div>
-			<div class="absolute inset-0 bg-gradient-to-r from-secondary/50 to-secondary/25"></div>
-			<div class="absolute inset-0 flex flex-col justify-between px-[1.875vw] pt-[2vw] text-accent">
-				<div class="flex flex-col gap-[0.2w]">
-					<span class="text-[1.5vw] font-bold leading-[1vw]">Welcome</span>
-					<span class="text-[0.875vw] font-semibold leading-[2.5vw]">Jump quickly into</span>
-				</div>
-				<div class="mt-[1vw] flex flex-col gap-[0.75vw]">
-					<div class="flex items-center gap-[1vw]">
-						<button class="btn btn-warning h-[2.5vw] min-h-max w-[2.5vw] rounded-[0.375vw] p-0">
-							<Chat class="text-surface-900 w-[1.25vw]" />
-						</button>
-						<span class="text-[1vw] font-bold">Forums</span>
-					</div>
-					<div class="flex items-center gap-[1vw]">
-						<button
-							class="bg-surface-50 btn btn-warning h-[2.5vw] min-h-max w-[2.5vw] rounded-[0.375vw] p-0"
-						>
-							<Recent class="text-surface-900 w-[1.25vw]" />
-						</button>
-						<span class="text-[1vw] font-bold">Last watched anime</span>
-					</div>
-					<div class="flex items-center gap-[1vw]">
-						<button
-							class="bg-surface-50 btn btn-warning h-[2.5vw] min-h-max w-[2.5vw] rounded-[0.375vw] p-0"
-						>
-							<Notification class="text-surface-900 w-[1.25vw]" />
-						</button>
-						<span class="text-[1vw] font-bold">Notifications</span>
-					</div>
-				</div>
-				<div class="mt-[0.4vw]">
-					<span class="text-[0.9vw] font-semibold leading-none">More</span>
-					<div class="mt-[0.75vw] flex gap-[0.9375vw]">
-						<button class="btn btn-accent h-[2.5vw] min-h-max w-[2.5vw] rounded-[0.375vw] p-0">
-							<Language class="text-surface-900 w-[1.25vw]" />
-						</button>
-						<button class="btn btn-accent h-[2.5vw] min-h-max w-[2.5vw] rounded-[0.375vw] p-0">
-							<Preference class="text-surface-900 w-[1.25vw]" />
-						</button>
-						<button class="btn btn-accent h-[2.5vw] min-h-max w-[2.5vw] rounded-[0.375vw] p-0">
-							<Moon class="text-surface-900 w-[1.25vw]" />
-						</button>
-						<button
-							class="bg-surface-50 btn btn-accent h-[2.5vw] min-h-max w-[2.5vw] rounded-[0.375vw] p-0"
-						>
-							<Settings class="text-surface-900 w-[1.25vw]" />
-						</button>
-					</div>
+		<div class="flex flex-col md:gap-[1vw]">
+			<span class="md:text-[1.35vw] text-accent font-bold">{$t("home.latest_episodes.title")}</span>
+			<div class="flex md:gap-[2vw] size-full">
+				<div class="w-full grid grid-cols-2 md:gap-[1.25vw]">
+					{#each latest_episodes as episode}
+						{#if episode.color_palette.length}
```

Comment:
{#if episode.color_palette.length}?
						
						Why is this looking like this?